_docs-2-0/development/client-properties.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Client Properties
+category: development
+order: 9
+---
+
+<!-- WARNING: Do not edit this file. It is a generated file that is copied from Accumulo build (from core/target/generated-docs) -->
+<!-- Generated by : org.apache.accumulo.core.conf.ClientConfigGenerate$Markdown -->
+
```

Comment:
Not sure if there is a good way to do this, but it seems like it would be nice if this pointed to the javadoc for building a connector... or maybe to the clients page.

Reply:
Add link to clients page in 2ee08b1229e8